api/src/main/java/bisq/api/http/service/auth/TokenRegistry.java

```diff
@@ -0,0 +1,63 @@
+package bisq.api.http.service.auth;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class TokenRegistry {
+
+    public static final long TTL = 30 * 60 * 1000;
+
+    private final TimeProvider timeProvider;
+    private final RandomStringGenerator randomStringGenerator;
+    private Map<String, Long> tokens = new HashMap<>();
+
+    public TokenRegistry() {
+        this(() -> UUID.randomUUID().toString(), System::currentTimeMillis);
```

Comment:
I think you have to use SecureRandom for ryptographically secure randomness.

Reply:
`UUID.randomUUID` uses `SecureRandom` by default.

Reply:
Ah ok.